bosch_thermostat_client/db/nsc_icom_gateway.json

```diff
@@ -0,0 +1,292 @@
+{
+    "04.07.03": {
+        "heatingCircuits": {
+            "hastates": [
+                {
+                    "haname": "off",
+                    "boschname": [
+                        "off"
+                    ]
+                },
+                {
+                    "haname": "heat",
+                    "boschname": [
+                        "manual"
+                    ]
+                },
+                {
+                    "haname": "heat_cool",
+                    "boschname": []
+                },
+                {
+                    "haname": "auto",
+                    "boschname": [
+                        "auto"
+                    ]
+                }
+            ],
+            "switchprogram": "/heatingCircuits/{}/switchPrograms/{}",
```

Comment:
you don't have switch programs. I think this has to be deleted and lib has to be changed to not rely on switchPrograms...

Reply:
I wrote to you on discord. We need to figure this switchPrograms out. It might be that you just don't have programs configured but those can be setup.